tests/test_jobfile.py

```diff
@@ -22,10 +23,11 @@ def test_job_file_submit_minimal(hq_env: HqEnv, tmp_path):
 def test_job_file_submit_maximal(hq_env: HqEnv, tmp_path):
     hq_env.start_server()
     hq_env.start_workers(3, cpus=4, args=["--resource", "gpus=[0,1]"])
+    os.mkdir("output")
     tmp_path.joinpath("job.toml").write_text(
         """
 name = "test-job"
-stream_log = "output.log"
+stream = "output"
```

Comment:
Documentation in `docs/jobs/jobfile.md` should be updated (`stream_log` ->` stream`) - it also contains a `--log` reference.